endpoint/test_endpoint.c

```diff
@@ -62,12 +60,12 @@ void test_endpoint(void) {
   gen_rand_trytes(next_addr, ADDR_LEN);
 
   endpoint_retcode_t ret = send_transaction_information(TEST_VALUE, TEST_MESSAGE, TEST_MESSAGE_FMT, TEST_TAG,
-                                                        TEST_ADDRESS, next_addr, test_key, TEST_DEVICE_ID, iv);
+                                                        TEST_ADDRESS, (char *)next_addr, test_key, TEST_DEVICE_ID, iv);
```

Comment:
does you apply formatter on this file? The format `(char *)` is different from other place.

Reply:
Follow the rules from clang format.